Tests/test_image.py

```diff
@@ -347,14 +347,24 @@ def test_effect_noise(self):
         # Arrange
         size = (100, 100)
         sigma = 128
+        # To reduce chance of randomly having the same value twice
```

Comment:
Instead of this approach, maybe we should do something like assert_image_similar, but with a minimum or expected difference.

Reply:
The image will be something like this: 
![image](https://user-images.githubusercontent.com/1324225/34160817-1ee76206-e4d7-11e7-974d-7140b944c25c.png)

For `assert_image_similar` the epsilon is 84-87. So a new `assert_image_different(self, a, b, minimum, msg=None)`?

Or how about taking 3 or more pixels and assert they're not all the same?

Reply:
There should be a pretty good relationship between sigma and the epsilon that we see, so checking more than three pixels will lead to a better test, and could potentially uncover sigma being treated incorrectly. 

However, this is a pretty low impact feature, so I'm not sure how much testing is really necessary.

Reply:
Replaced it with an `assert_not_all_same([p0, p1, p2, p3, p4])`